src/structlog/processors.py

```diff
@@ -586,7 +586,7 @@ def __call__(
         return event_dict
 
 
-def _figure_out_exc_info(v: Any) -> ExcInfo:
+def _figure_out_exc_info(v: Any) -> Union[ExcInfo, Tuple[None, None, None]]:
```

Comment:
The commit hook wants me to write `ExcInfo | tuple[None, None, None]` but this will break on older Python versions.  For Typed Settings, I configured

```toml
[tool.ruff]
target-version = "py38"
```

to fix this.

Reply:
this shouldn't break with `from __future__ import annotations`